slick-testkit/src/test/scala/slick/test/codegen/CodeGenRoundTripTest.scala

```diff
@@ -36,12 +36,12 @@ class CodeGeneratorRoundTripTest(val tdb: JdbcTestDB) extends DBTest {
       SelfRef.result,
       {
         // Testing table larger 22 columns
-        val oData = 0L :: 11 :: 12 :: 13 :: 14 :: 15 :: 16 :: 21 :: 22 :: 23 :: 24 :: 25 :: 26 :: 31 :: 32 :: 33 :: 34 :: 35 :: 36 :: 41 :: 42 :: 43 :: 44 :: 45 :: 46 :: 51 :: 52 :: 53 :: 54 :: 55 :: 56 :: 61 :: 62 :: 63 :: 64 :: 65 :: 66 :: HNil
+        val oData = LargeRow(0L, 11 , 12 , 13 , 14 , 15 , 16 , 21 , 22 , 23 , 24 , 25 , 26 , 31 , 32 , 33 , 34 , 35 , 36 , 41 , 42 , 43 , 44 , 45 , 46 , 51 , 52 , 53 , 54 , 55 , 56 , 61 , 62 , 63 , 64 , 65 , 66)
```

Comment:
Hmmm.. I'm not sure how this will be possible. I removed the HList test because in the previous implementation all rows > 22 columns were returned as HList, With the new setup the HList and case class test cant exist at the same time without causing a compile error.